ios/Podfile

```diff
@@ -1,14 +1,35 @@
-platform :ios, '11.0'
-
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+platform :ios, '12.4'
+# install! 'cocoapods', :deterministic_uuids => false
+
+production = ENV["PRODUCTION"] == "1"
+
+# We need to enable codegen discovery to fix the issue of these files not being generated properly:
+# ios/build/generated/ios/FBReactNativeSpec/FBReactNativeSpec-generated.mm
+# ios/build/generated/ios/FBReactNativeSpec/FBReactNativeSpec.h
+# more details here : https://github.com/status-im/status-mobile/issues/15911
+# TODO: remove this flag once you upgrade react-native above version 0.70
+ENV['USE_CODEGEN_DISCOVERY'] = "1"
+
 abstract_target 'Status' do
   # Pods for StatusQuo
   config = use_native_modules!
 
+  # Flags change depending on the env values.
+  flags = get_default_flags()
+
   use_react_native!(:path => config[:reactNativePath],
-                    :hermes_enabled => false)
+                    # to enable hermes on iOS, change `false` to `true` and then install pods
+                    :production => production,
+                    :hermes_enabled => flags[:hermes_enabled],
+                    :fabric_enabled => flags[:fabric_enabled],
+                    :use_codegen_discovery => true,
+                    # :flipper_configuration => FlipperConfiguration.enabled,
+                    # An absolute path to your application root.
+                    :app_path => "#{Pod::Config.instance.installation_root}/.."
+  )
```

Comment:
If we are putting the closing bracket on the same indent level as function call the arguments should have a normal 2 space indent:
https://github.com/rubocop/ruby-style-guide#no-double-indent